src/flag_gems/__init__.py

```diff
@@ -191,6 +191,7 @@ def enable(lib=aten_lib, unused=None):
             ("diag", diag, Autograd.disable),
             ("diag_embed", diag_embed, Autograd.disable),
             ("diagonal_backward", diagonal_backward, Autograd.disable),
+            ("count_nonzero",count_nonzero,Autograd.disable),
```

Comment:
Please `pip install pre-commit`, and use `pre-commit` to format codes.

Reply:
ok